compiler/one-cmds/one-quantize

```diff
@@ -345,6 +361,31 @@ def _verify_arg(parser, args):
             parser.error(
                 'The same number of src_tensor_name and dst_tensor_name should be given.')
 
+    # Check calibration parameters
+    if oneutils.is_valid_attr(args, 'mode'):
+        if getattr(args, 'mode') == 'percentile':
+            # Check dtype
+            try:
+                min_percentile = float(getattr(args, 'min_percentile'))
+            except ValueError:
+                parser.error('min_percentile must be float')
+            try:
+                max_percentile = float(getattr(args, 'max_percentile'))
+            except ValueError:
+                parser.error('max_percentile must be float')
+        elif getattr(args, 'mode') == 'moving_average':
+            # Check dtype
+            try:
+                moving_avg_batch = int(getattr(args, 'moving_avg_batch'))
+            except ValueError:
+                parser.error('moving_avg_batch must be integer')
+            try:
+                moving_avg_const = float(getattr(args, 'moving_avg_const'))
```

Comment:
Also, I am not sure `float` will raise error when `moving_avg_const` is integer.

Reply:
Integer value is ok for `moving_avg_const`.

Reply:
Hmm.. there is no check for `moving_avg_const` has to be [0,1]. Let me add the check in `record-minmax` in the next PR.

Reply:
I see. By `float`, any number is acceptable (including `int`). But it is okay. You use `float` to check whether it is number.